### PR TITLE
Fix/planet env storybook

### LINKED ISSUE #197: PLANET_API_KEY "placeholder" missing in .env.example
When running the storybook, the `PLANET_API_KEY` is mandatory, but the "placeholder" for it isn't present in the `.env.example` file, suggesting it's not mandatory.

### PR DESCRIPTION
**Pull request by guss84**
fixes #197 

Also noticed that one of the stories had a bug caused by an api change to the `tileCoord` object

### CODE REVIEW

File: stories/wmts.planet.stories.js [hunk 1]
```diff
@@ -95,7 +94,6 @@ export const getMapWTmsLayersFactory = () => {
       width: 256,
```
Comment (zcernigoj) [inline stories/wmts.planet.stories.js:94]: There's probably a typo in the name of this story - `getMapWTmsLayersFactory` -> should be `getMapWmtsLayersFactory`

File: stories/wmts.planet.stories.js [hunk 2]
```diff
@@ -37,7 +37,6 @@ export const getMapBbox = () => {
...
       height: 256,
       format: MimeTypes.JPEG,
-      zoom: 14,
```
Comment (zcernigoj) [inline stories/wmts.planet.stories.js:40]: If I understand this correctly, there are 2 ways of setting the "location" 
- bbox (and no zoom is required)
- tileCoord

The **bbox and no zoom** conclusion is based on no `zoom` being present in `src/layer/WmtsLayer.ts`.
If this conclusion is correct, should the `zoom` also be removed from:
- `GetMapParams` in `src/layer/const.ts`
- `WmtsLayer.getMap uses bbox` test in `src/layer/__tests__/WmtsLayer.ts`
-

File: stories/wmts.planet.stories.js (hunk 1 above)
Comment (guss84) [reply stories/wmts.planet.stories.js:94]: Fixed, thanks

File: stories/wmts.planet.stories.js (hunk 2 above)
Comment (guss84) [reply stories/wmts.planet.stories.js:40]: Fixed. Thanks for the thorough check.

File: stories/wmts.planet.stories.js (hunk 1 above)
Comment (zcernigoj) [reply stories/wmts.planet.stories.js:94]: :+1: Resolving

File: stories/wmts.planet.stories.js (hunk 2 above)
Comment (zcernigoj) [reply stories/wmts.planet.stories.js:40]: Looks good now. Resolving

**Review by zcernigoj — APPROVED** [review]

**Pull request merged**